examples/todo/src/controllers/todo.controller.ts

```diff
@@ -27,6 +32,13 @@ export class TodoController {
     if (!todo.title) {
       throw new HttpErrors.BadRequest('title is required');
     }
+
+    if (todo.remindAtAddress) {
+      // TODO(bajtos) handle "address not found"
+      const geo = await this.geoService.geocode(todo.remindAtAddress);
+      todo.remindAtGeo = `${geo[0].y},${geo[0].x}`;
```

Comment:
This is a great question! For reference, `y` is `latitude` and `x` is longitude. 

As far as I can tell from searching around, there is no wide consensus on the right order :(

In StrongLoop, we were always using `lat,lng` order to be compatible with Google Maps API, since that's the API most people seem to be using and be familiar with these days .

https://stackoverflow.com/q/7309121/69868
> - EPSG:4326 specifically states that the coordinate order should be latitude, longitude. Many software packages still use longitude, latitude ordering. This situation has wreaked unimaginable havoc on project deadlines and programmer sanity.
> - The prefered order is by convention latitude, longitude. This was presumably standardized by the International Maritime Organization as reported here. Google also uses this order in its Maps and Earth. I remember this order by thinking of alphabetic order of latitude, longitude.

BUT:

> The correct order is longitude, latitude, in virtually all professional GIS applications, as it is in conventional math (ie, f(x ,y, z)). The GeoJSON standard is fairly typical and succinct:
> > The order of elements must follow x, y, z order
(easting, northing, altitude for coordinates in a 
projected coordinate reference system, or longitude,
latitude, altitude for coordinates in a geographic
coordinate reference system).
>
> The same is true of the primary Open Geospatial Consortium standards (WKT and WKB, and extensions like EWKB). Likewise Google may output the order in Lat/Lon to make it more familiar to users who grew up with that custom (ie from navigation standards like IMO, rather than computational ones.) But the KML standard itself is like virtually all other GIS systems:

https://gis.stackexchange.com/q/7379
> We are used to seeing latitude and longitude marked on paper maps, tempting us to think of these as planar (Cartesian) coordinates. However, they are not; the paper (planar) map is a projection of spherical coordinates, and spherical coordinates are generally written as radius, inclination, and azimuth (at least in physics.) In fact, the order radius, inclination, azimuth is codified in ISO 31-11. Geographers don't need the radius (or to the extent they do, they use elevation/altitude, which is the deviation from the nominal radius of the earth), so we just have inclination (latitude) and azimuth (longitude). From this perspective, latitude/longitude is perfectly rational.

Reply:
I added a comment explaining the order and pointing to those stackexchange/stackoverflow pages.

To be honest, I don't really mind which order we use. If you think we should switch to `lng,lat`, then please let me know and I'll make that change in a follow-up pull request. (You can also send that PR yourself if you like.)

Reply:
The convention I'm familiar with / expect myself is `lat, long` but it was the `x,y` that threw me ... but we don't exactly have control over that as it's coming from an external service. I'm ok with the changes as they currently stand. Thanks.